pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -0,0 +1,50 @@
+package configobservercontroller
+
+import (
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	kubeinformers "k8s.io/client-go/informers"
+
+	configinformers "github.com/openshift/client-go/config/informers/externalversions"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+
+	"github.com/openshift/cluster-openshift-controller-manager-operator/pkg/operator/configobservation"
+	"github.com/openshift/cluster-openshift-controller-manager-operator/pkg/operator/configobservation/builds"
+	"github.com/openshift/cluster-openshift-controller-manager-operator/pkg/operator/configobservation/deployimages"
+	"github.com/openshift/cluster-openshift-controller-manager-operator/pkg/operator/configobservation/images"
+)
+
+type ConfigObserver struct {
+	*configobserver.ConfigObserver
+}
+
+// NewConfigObserver initializes a new configuration observer.
+func NewConfigObserver(
+	operatorClient v1helpers.OperatorClient,
+	configInformers configinformers.SharedInformerFactory,
+	kubeInformersForOperatorNamespace kubeinformers.SharedInformerFactory,
+	eventRecorder events.Recorder,
+) *ConfigObserver {
+	c := &ConfigObserver{
+		ConfigObserver: configobserver.NewConfigObserver(
+			operatorClient,
+			eventRecorder,
+			configobservation.Listers{
+				ImageConfigLister: configInformers.Config().V1().Images().Lister(),
+				ImageConfigSynced: configInformers.Config().V1().Images().Informer().HasSynced,
+				BuildConfigLister: configInformers.Config().V1().Builds().Lister(),
+				BuildConfigSynced: configInformers.Config().V1().Builds().Informer().HasSynced,
+				ConfigMapLister:   kubeInformersForOperatorNamespace.Core().V1().ConfigMaps().Lister(),
+				ConfigMapSynced:   kubeInformersForOperatorNamespace.Core().V1().ConfigMaps().Informer().HasSynced,
```

Comment:
two items:
1. Pass in a `ResourceSyncer`?
2. Pre-run cache sync for the operator config object?

See [cluster-openshift-apiserver-operator](https://github.com/openshift/cluster-openshift-apiserver-operator/blob/8aa4775bcff1c0e3d62606006cef70e4fda3d8f9/pkg/operator/configobservation/configobservercontroller/observe_config_controller.go#L46)

Reply:
1) we don't have a resourcesyncer to pass because we don't have any resources we are syncing across namepaces.

2) i didn't see an obvious use/value in the pre cache sync given that we need to check it in the observers.